src/main/scripts/org/reaktivity/specification/nukleus/http/streams/rfc7230/architecture/request.and.response/raw/client.rpt

```diff
@@ -0,0 +1,38 @@
+#
+# Copyright 2016-2017 The Reaktivity Project
+#
+# The Reaktivity Project licenses this file to you under the Apache License,
+# version 2.0 (the "License"); you may not use this file except in compliance
+# with the License. You may obtain a copy of the License at:
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+#
+
+# network -> source (this script) -> http -> target (cooked)
+# Write to http/source, read from source/http
+
+property target "http"
+property newSourceInputRef ${nuklei:newReferenceId()} # external scope
+
+connect "nukleus://streams/${target}"
+    nukleus:option source "source"
+    nukleus:option route ${newSourceInputRef}
+    nukleus:option window 8192
+    nukleus:option transmission "duplex"
```

Comment:
Ideally it would be possible to parameterize the transport URI and options, something like:
```
connect ${connectURI} ${connectOptions}
```
so the script can be used against any HTTP server, for example cutting WebSocket gateway.